### PR TITLE
Add shunt compensator modification

### PR DESCRIPTION
**Pull request by YenguiSeddik**

### CODE REVIEW

File: src/components/dialogs/network-modifications/shunt-compensator/characteristics-pane/characteristics-form-utils.js [hunk 1]
```diff
@@ -15,17 +15,21 @@ import {
...
         .number()
         .nullable()
         .when([CHARACTERISTICS_CHOICE], {
-            is: CHARACTERISTICS_CHOICES.SUSCEPTANCE.id,
+            is: (characteristicsChoice) =>
+                characteristicsChoice ===
+                    CHARACTERISTICS_CHOICES.SUSCEPTANCE.id && !isModification,
             then: (schema) => schema.required(),
         }),
     [SHUNT_COMPENSATOR_TYPE]: yup.string().when([CHARACTERISTICS_CHOICE], {
```
Comment (Meklo) [inline src/components/dialogs/network-modifications/shunt-compensator/characteristics-pane/characteristics-form-utils.js:29]: Shouldn't this field be nullable it's a modification situation ?

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: ok
console: no warning
tests:
1) Issues:
- "Name", "susceptance" and "A at nominal voltage" fields must be nullable (have a cross to empty the field)
- when we create a modification, and just select an Id, we can validate the form wihout any modification.
In this case we create a modification in db with a change on "voltage_level_id".
We can realize the modification, but there is nothing to do (no logs).

2) remarks:
- we should use the same label in creation and modification sub-menus (same in EN but different in FR)

**Review by dbraquart — APPROVED** [review]
code: ok
console: ok
tests: almost ok
- we still can validate an "empty" modification, but I think it can be resolve later by an existing US.

**Review by sBouzols — APPROVED** [review]
Code Review OK
Tests OK
Console Warning Check OK

**Review by EstherDarkish — COMMENTED** [review]
Found an edge case that is maybe a bug

**Review by EstherDarkish — APPROVED** [review]
